### ISSUE TITLE
Feature: library browser column mode

### ISSUE THREAD

**Issue by raffraffraff** [clementine-player/Clementine#6771]
### System information

Please provide information about your system and the version of Clementine used.

- Operating System: Ubuntu 20.04
- Clementine version: 1.4 rc1

### Expected behaviour / actual behaviour

The Library browser needs a column layout with user-configurable columns (just like grandaddy Amarok 1.4 had). Casually browsing through a 200gb collection using a tree-mode is horrendous, and it's the reason why I continued compiling my own Amarok 1.4 for years after the Amarok deverlopers lost their minds with 2.0, and none of the forks (eg: Clementine)  implemented that Library view mode.  

### Steps to reproduce the problem (only for bugs)

Browse through an gigantic tree-of-songs and use the search. Except, what if I don' know the _exact_ song I'm looking for? What if I know fuzzy details, like "it came out sometime in the 70s, has a 4+ rating is in the genre psychadelic rock"? With Amarok 1.4 and MusicBee, the library browser lets me choose columns (Artist, Title, Album, Rating, Genre, Year, LastFM <3) and I can use the search as a basic filter, type "psychadelic", sort by rating, scroll to the year and boom, there's the tune I want, with a LastFM <3 on it. I also get a nice tabluar list of similar tunes, ordered by rating and what-do-you-know, it's the makings of a great playlist. but changing column ordering and tweaking the search string, I can generate amazing playlists super quickly. ...Or I could stare at the tree struggling to think of the tune I want.

Instead of giving all this amazing context when I'm looking for the tune I want (ie: the hard part) I get it when it's already in a playlist (great for sure, but come on, if it's right there already all metadata is one click away, so it's not a huge deal)

**Comment by studiosusmed** [comment]
i strongly agree. especially now that guayadeque is discontinued, some user base might be willing to turn over, if clementine has a more modular library design.
